big_tests/tests/mam_SUITE.erl

```diff
@@ -2974,6 +2976,24 @@ check_user_exist(Config) ->
   %% cleanup
   ok = escalus_ejabberd:rpc(ejabberd_auth, remove_user, [AdminU, AdminS]).
 
+decode_compact_uuid(_Config) ->
```

Comment:
I'd suggest to create a "small" suite for MAM for this case (e.g. `[repo]/test/mam_SUITE.erl`), as it does RPC calls and does not generate any XMPP traffic.

Reply:
`[repo]/test/mod_mam_SUITE.erl`.

I would love to not have name duplicates.

Reply:
So actually it should be `mod_mam_utils_SUITE` :)